### PR TITLE
[Kucoin][OKX] add get_account_id

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/956

### CODE REVIEW

**Review by Herklos — APPROVED** [review]
:+1:

File: Trading/Mode/staggered_orders_trading_mode/staggered_orders_trading.py [hunk 1]
```diff
@@ -285,12 +291,21 @@ async def create_order(self, order_data, current_price, symbol_market):
...
                 if selling:
-                    if trading_api.get_portfolio_currency(self.exchange_manager, currency).available < order_quantity:
+                    available = trading_api.get_portfolio_currency(self.exchange_manager, currency).available
+                    if available < order_quantity:
+                        self.logger.error(f"Skipping order creation: not enough {currency}: available: {available}, "
+                                          f"required: {order_quantity}")
                         return []
-                elif trading_api.get_portfolio_currency(self.exchange_manager, market).available < order_quantity * order_price:
+                elif market_available < order_quantity * order_price:
+                    self.logger.error(
```
[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply Trading/Mode/staggered_orders_trading_mode/staggered_orders_trading.py:302]: yes, this should not happen. If it does, there is an algo issue when scheduling orders

File: Trading/Mode/staggered_orders_trading_mode/staggered_orders_trading.py [hunk 2]
```diff
@@ -819,6 +836,100 @@ def _check_params(self):
...
+                    self.logger.error(f"Timeout while waiting for rebalance marker order fill, order {created_order}")
+        if created_order.is_open() and created_order.state is not None:
+            self.logger.error(f"Rebalance marker order is still open, order {created_order}")
+        else:
+            self.logger.info("Successfully rebalanced funds after missed mirror orders.")
+
+    def _find_missing_mirror_order_fills(self, sorted_trades, missing_orders):
+        trades_with_missing_mirror_order_fills = []
+        price_increment = self.flat_spread - self.flat_increment
+        price_window = self.flat_increment / decimal.Decimal(4)
```
[the remark this answers is not in the document]
Comment (GuillaumeDSM) [reply Trading/Mode/staggered_orders_trading_mode/staggered_orders_trading.py:911]: No, here the idea is to have a window to account to exchange price rounding, 4 is a constant here

[the remark this answers is not in the document]
Comment (Herklos) [reply Trading/Mode/staggered_orders_trading_mode/staggered_orders_trading.py:911]: ok

**Review by Herklos — APPROVED** [review]

**Pull request merged**